docker/docker-compose.yaml

```diff
@@ -0,0 +1,143 @@
+version: '3'
+
+services:
+  veritable-demo-init:
+    build:
+      context: .
+      dockerfile: Dockerfile.init
+    container_name: veritable-demo-init
+    networks:
+      - von
+  cloudagent-issuer:
+    image: bcgovimages/aries-cloudagent:py36-1.16-1_0.7.3
+    container_name: veritable-cloudagent-issuer
+    command: 'start --inbound-transport http 0.0.0.0 8000 --inbound-transport ws 0.0.0.0 8001 --admin 0.0.0.0 8002'
+    restart: on-failure
+    environment:
```

Comment:
that is a lot of variables, maybe abstract into a file? or have something like default config for some like

```
      - ACAPY_AUTO_PROVISION=true
      - ACAPY_WALLET_TYPE=indy
      ...
      - ACAPY_AUTO_ACCEPT_INVITES=true
      - ACAPY_AUTO_ACCEPT_REQUESTS=true
      - ACAPY_AUTO_STORE_CREDENTIAL=true
      - ACAPY_PRESERVE_EXCHANGE_RECORDS=true
```
Looks like they are being shared across all personas

Reply:
not a bad idea, I can do that. There are some differences between personas but I'll pull out the common stuff